### PR TITLE
Add recover-broker-info command details

### PR DESCRIPTION
**Pull request by Jaanki**
Epic: https://github.com/submariner-io/enhancements/issues/143

### CODE REVIEW

File: src/content/operations/deployment/subctl/_index.en.md [hunk 1]
```diff
@@ -539,6 +539,13 @@ The following steps are performed:
 | `--namespace` `<string>`         | Namespace in which Submariner is installed (default `submariner-operator`)
 | `--yes`                          | Automatically answer yes to confirmation prompt
 
+### `recoverbroker-info.subm`
```
Comment (tpantelis) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
### `recover-broker-info`
```

File: src/content/operations/deployment/subctl/_index.en.md [hunk 2]
```diff
@@ -539,6 +539,13 @@ The following steps are performed:
...
+
+`subctl recover-broker-info`
```
Comment (maayanf24) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
`subctl recover-broker-info [flags]`
```

File: src/content/operations/deployment/subctl/_index.en.md [hunk 3]
```diff
@@ -539,6 +539,13 @@ The following steps are performed:
...
+
+This command recovers lost `broker-info.subm` file. Optionally, a cluster where `Broker` is installed can be specified via `--context`
+or `--kubeconfig` flag. The command also works if no context is specified.
```
Comment (maayanf24) [inline src/content/operations/deployment/subctl/_index.en.md]: Please add all the flags in a table

**Review by maayanf24 — CHANGES_REQUESTED** [review]

File: src/content/operations/deployment/subctl/_index.en.md (hunk 3 above)
Comment (tpantelis) [inline src/content/operations/deployment/subctl/_index.en.md]:
```suggestion
This command recovers a lost `broker-info.subm` file.

#### `recover-broker-info` flags

| Flag                             | Description
|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
| `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
| `--context ` `<string>`      | kubeconfig context of a cluster where Submariner is installed
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/operations/deployment/subctl/_index.en.md [hunk 4]
```diff
@@ -556,6 +556,19 @@ The following steps are performed:
...
+
+`subctl recover-broker-info [flags]`
+
+This command recovers a lost `broker-info.subm` file.
+
+#### `recover-broker-info` flags
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--kubeconfig` `<string>`        | Absolute path(s) to the kubeconfig file(s)
```
Comment (dfarrell07) [inline src/content/operations/deployment/subctl/_index.en.md:572]: These are covered in https://github.com/submariner-io/submariner-website/pull/910/files#diff-0c05bfa14f9a42fbb974a98180586148ba00100482b3222225bc82ed53d7e2ddR35, I don't think the should be duplicated here.

Comment (dfarrell07) [reply src/content/operations/deployment/subctl/_index.en.md:572]: Although I see this is also duplicated for many commands. Can you raise an issue to de-dup this @Jaanki?

**Review by skitt — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]

File: src/content/operations/deployment/subctl/_index.en.md (hunk 4 above)
Comment (Jaanki) [reply src/content/operations/deployment/subctl/_index.en.md:572]: Raised https://github.com/submariner-io/submariner-website/issues/914

**Review by tpantelis — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Pull request merged**